tests/test_online_replacing.py

```diff
@@ -232,3 +237,37 @@ def testReadMultiple(self):
         response = requests.get("http://example.com")
         self.assertIn("This domain is for use", response.text)
         self.assertAlmostEqual(0.9974949866040544, sin_output, delta=0.0005)
+
+
+new_cassette = Cassette()
+
+
+@apply_decorator_to_class_methods(
+    replace_module_match(
+        what="math.sin", decorate=Simple.decorator_plain(), cassette=new_cassette
+    )
+)
+@apply_decorator_to_class_methods(record_requests(cassette=new_cassette))
```

Comment:
What about having a shortcut for that? Since this will be used very often, it makes sense to have it.

```python
def record_requests_for_all_methods(cassette: Optional[Cassette] = None):
    return apply_decorator_to_class_methods(record_requests(cassette=cassette))
```

(Or ideally, add that trick to be able to use both `@record_requests_for_all_methods` and `@record_requests_for_all_methods()`.)

Reply:
we can add it, but I don't see so big benefit there :-)